commands/init.ts

```diff
@@ -122,8 +127,12 @@ exports.handler = async options => {
     process.exit(EXIT_CODES.ERROR);
   }
 
-  trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
+  if (!disableTracking) {
+    await trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
+  }
+
   createEmptyConfigFile({ path: configPath });
```

Comment:
If we wanted to spend some more time on this, we could update this method to set the `allowUsageTracking` to false when the flag is a part of the command.  It seems like since the main reason we are adding this is to disable the tracking from ATs this might be overkill.  And then it might be strange if we support one of the `hs config set` items and not the others.  That's why I went with this approach